single_stage_detector/ssd/README.md

```diff
@@ -1,287 +1,27 @@
-# 1. Summary
-Single Shot MultiBox Detector (SSD) is an object detection network. For an input image, the network outputs a set of bounding boxes around the detected objects, along with their classes. For example:
+# Training
```

Comment:
Can you please provide the instructions here to build the Docker image?

Reply:
See my comment [here](https://github.com/mlcommons/training/pull/517/files/b8ee39d86041d583f4f86ac182e49ffdebed39a9#diff-fcd2cc56acbb29907bb42227634c88e033c0f27ef387ec7cf3dd6b2fde3776f9)